### PR TITLE
SEO Improvements

### PR DESCRIPTION
**Pull request by Hassaan-Qaisar**

### CODE REVIEW

File: next-sitemap.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// next-sitemap.config.js
+
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  siteUrl: 'https://chained-assets.vercel.app',
```
Comment (RobinNagpal) [inline next-sitemap.config.js]: should it be `https://chained-assets.vercel.app` or `https://chainedassets.com` ?

File: next-sitemap.config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// next-sitemap.config.js
+
+/** @type {import('next-sitemap').IConfig} */
```
Comment (RobinNagpal) [inline next-sitemap.config.js:3]: Why this commented out lines?

File: public/robots.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+
+Sitemap: https://chained-assets.vercel.app/sitemap.xml
```
Comment (RobinNagpal) [inline public/robots.xml]: Update url? `https://chained-assets.vercel.app/sitemap.xml`

File: src/app/landscape/page.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import { Metadata } from 'next'
+import type { Viewport } from 'next'
+
+export const viewport: Viewport = {
+  width: 'device-width',
+  initialScale: 1,
+}
+
+export const metadata: Metadata = {
+  metadataBase: new URL('https://chained-assets.vercel.app/landscape'),
```
Comment (RobinNagpal) [inline src/app/landscape/page.tsx]: Extract out this common content in this and next file in a common utility file/function?

File: next-sitemap.config.js (hunk 1 above)
Comment (Hassaan-Qaisar) [reply next-sitemap.config.js]: it should be https://chainedassets.com. at that time i donot know that we also deployed this website on another url as well

File: public/robots.xml (hunk 3 above)
Comment (Hassaan-Qaisar) [reply public/robots.xml]: yes. it should be updated

File: src/app/landscape/page.tsx (hunk 4 above)
Comment (Hassaan-Qaisar) [reply src/app/landscape/page.tsx]: sure. it should be useful for code reusing

File: next-sitemap.config.js (hunk 2 above)
Comment (Hassaan-Qaisar) [reply next-sitemap.config.js:3]: This is a JSDoc comment that provides type information for the module.exports object.

**Pull request merged**